nixos/modules/services/printing/cups-pdf.nix

```diff
@@ -0,0 +1,205 @@
+{ config, lib, pkgs, ... }:
+
+let
+
+  inherit (lib.attrsets) filterAttrs getAttr mapAttrs mapAttrsToList;
+  inherit (lib.meta) getExe;
+  inherit (lib.modules) mkAfter mkDefault mkIf;
+  inherit (lib.options) literalExpression mkEnableOption mkOption;
+  inherit (lib.strings) concatStrings;
+  inherit (lib.trivial) pipe;
+  inherit (lib.types) attrsOf int lines nullOr oneOf package path singleLineStr str submodule;
```

Comment:
```suggestion
```
please use with lib or prefix the the functions with lib. Maintaining that list goes not have a real benefit.

Reply:
I try to avoid `with` whereever possible (similar to wildcard imports in Python), so I added `lib.` prefixes.